client/reader/site-stream/index.jsx

```diff
@@ -27,55 +34,59 @@ function checkForRedirect( site ) {
 	}
 }
 
-const SiteStream = React.createClass( {
+class SiteStream extends React.Component {
 
-	getDefaultProps: function() {
-		return { showBack: true };
-	},
+	static propTypes = {
+		siteId: React.PropTypes.number.isRequired
```

Comment:
There were no prop types specified on the component before, so this is a new one. I'll fix!